### PR TITLE
Add Share page permission

### PR DESCRIPTION
**Pull request by kurtw**
Adds Share page permission to the V1 CFGOVPage model and then adds it as an option for a Group.

@kave 
@rosskarchner

### CODE REVIEW

File: cfgov/v1/migrations/0012_share_perms.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    # Create share permission
+    share_permission = Permission.objects.create(
+        content_type=v1_content_type,
+        codename='share_page',
+        name='Can share pages'
+    )
+
+    # Assign it to Editors and Moderators groups
+    for group in Group.objects.filter(name__in=['Editors', 'Moderators']):
```
Comment (kave) [inline cfgov/v1/migrations/0012_share_perms.py]: we only have 2 default groups, so filter is unnecessary

Comment (kurtw) [inline cfgov/v1/migrations/0012_share_perms.py]: I guess that's true. This is the code they had for something else that I just took :) We can just grab them all? Keeping in mind any other groups will have it.

Comment (kave) [inline cfgov/v1/migrations/0012_share_perms.py]: lol gotcha, yea i figure it doesn't hurt if they add more default groups in future releases

**Comment by kave** [comment]
:+1:

**Pull request merged**